contracts/RelayerHub.sol

```diff
@@ -56,13 +56,6 @@ contract RelayerHub is IRelayerHub, System, IParamSubscriber{
     alreadyInit = true;
   }
 
-  function register() external payable noExist onlyInit notContract noProxy{
```

Comment:
You have to change this part in template too

Reply:
Let us add a templated.